### PR TITLE
feat: add sponsors-section

### PR DESCRIPTION
**Pull request by eazycodinz**
I just added a new feature, the sponsors-section to the project. i also deleted the team image from the image folder.

### CODE REVIEW

File: components/sections/SponsorsSection/SponsorsSection.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+export function SponsorsSection() {
+  return (
+    <section className="container max-w-screen-2xl mx-auto">
+      <div>
+        <div>
+          <h2 className="font-heading md:text-4xl text-2xl text-center pb-10">
+            Sponsors and Partners
+          </h2>
+        </div>
+        <div className="grid grid-cols-3 gap-4 justify-items-center md:px-20 px-10">
```
Comment (0xJamin) [inline components/sections/SponsorsSection/SponsorsSection.tsx:13]: use `grid-cols-2` for tablets and mobile views

**Review by 0xJamin — CHANGES_REQUESTED** [review]

File: components/sections/SponsorsSection/SponsorsSection.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Image3 from "@/public/images/recapimage3.png";
+import Image4 from "@/public/images/recapimage4.png";
+export function SponsorsSection() {
```
Comment (0xJamin) [inline components/sections/SponsorsSection/SponsorsSection.tsx:4]: `SponsorSection`

File: components/sections/SponsorsSection/SponsorsSection.tsx (hunk 1 above)
Comment (eazycodinz) [reply components/sections/SponsorsSection/SponsorsSection.tsx]: resolved

File: components/sections/SponsorsSection/SponsorsSection.tsx (hunk 2 above)
Comment (eazycodinz) [reply components/sections/SponsorsSection/SponsorsSection.tsx]: resolved

Comment (eazycodinz) [reply components/sections/SponsorsSection/SponsorsSection.tsx]: renamed the section from SponsorsSection to SponsorSection

File: components/sections/SponsorsSection/SponsorsSection.tsx (hunk 1 above)
Comment (eazycodinz) [reply components/sections/SponsorsSection/SponsorsSection.tsx]: fixed the responsiveness of the SponsorSection by adding grid-cols-2 for tablets and mobile views.

File: components/sections/SponsorSection/SponsorSection.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    <section className="container max-w-screen-2xl mx-auto">
+      <div>
+        <div>
+          <h2 className="font-heading md:text-4xl text-2xl text-center pb-10">
+            Sponsors and Partners
+          </h2>
+        </div>
+        <div className="grid md:grid-cols-3 grid-cols-2 gap-4 justify-items-center md:px-20 px-10">
+          <div>
+            <img className="md:w-52 w-36 md:h-12 " src={Image3.src} alt="" />
```
Comment (0xJamin) [inline components/sections/SponsorSection/SponsorSection.tsx:15]: @eazycodinz add some padding and margin between the sponsors

**Review by 0xJamin — CHANGES_REQUESTED** [review]

**Review by 0xJamin — CHANGES_REQUESTED** [review]

**Review by 0xJamin — APPROVED** [review]
Looks good @eazycodinz  well done

**Pull request merged**